server.js

```diff
@@ -59,7 +59,16 @@ const app = express();
 app.use(session({
   secret: 'foo',
   resave: false,
-  saveUninitialized: true
+  saveUninitialized: true,
+  // This example project can be used to build a plugin using the Plugin Framework.
+  // Note that this example project's particular cookie technique will only work in Chromium-based browsers e.g.
+  // - Google Chrome
+  // - newer versions of Microsoft Edge
+  // This cookie technique is NOT recommended for use in production systems.
```

Comment:
Actually these are recommended settings. However these cookies are going to be blocked by Chromium based browsers in the future and are already blocked by Safari and FireFox by default.

Safari can be made to work by disabling the "Prevent cross-site tracking" option.
<img width="774" alt="Screen Shot 2020-08-28 at 10 45 26 AM" src="https://user-images.githubusercontent.com/1247639/91586920-b0ee0880-e91b-11ea-9693-02c44f238321.png">

Reply:
One note - Safari can be made to work for the developer, but this isn't a solution for production customers.

Reply:
Addressed (hopefully) in https://github.com/Banno/consumer-api-openid-connect-example/pull/17/commits/2b57fda5b2cec92d572f669d6164d0c3bd361ae0